src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1060,4 +1060,10 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Hyperlink_Owner" xml:space="preserve">
     <value>Owner</value>
   </data>
+  <data name="Label_InstalledVulnerabilitiesCount" xml:space="preserve">
+    <value>{0} vulnerable versions installed</value>
+  </data>
+  <data name="Label_InstalledVulnerabilitiesCountSingle" xml:space="preserve">
+    <value>{0} vulnerable version installed</value>
+  </data>
```

Comment:
It might be a good idea to add a comment for the translators to know that the {0} token is a numeric value.